parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -22,7 +22,7 @@ installDeps() {
     
     dnf_makecache || exit $ERR_APT_UPDATE_TIMEOUT
     dnf_update || exit $ERR_APT_DIST_UPGRADE_TIMEOUT
-    for dnf_package in ca-certificates check-restart cifs-utils cloud-init-azure-kvp conntrack-tools cracklib dnf-automatic ebtables ethtool fuse git inotify-tools iotop iproute ipset iptables jq kernel-devel logrotate lsof nmap-ncat nfs-utils pam pigz psmisc rsyslog socat sysstat traceroute util-linux xz zip blobfuse2 nftables iscsi-initiator-utils; do
```

Comment:
Curious about why `git` and `kernel-devel` were added in the first place

Reply:
Git was a copy from the Ubuntu list of packages from the start of the Mariner configs. Kernel-devel was added to support Microsoft Defender for Containers, but is no longer required for it to function.